### PR TITLE
Upgrade gent_styleguide to latest version (3.0.0-beta8)

### PR DESCRIPTION
**Pull request by hstandaert**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: source/sass/31-molecules/status-message/_status-message.scss [hunk 1]
```diff
@@ -1,4 +1,9 @@
 .messages {
   background-image: none;
   box-shadow: none;
+
+  // @TODO: remove this when the styleguide version is upgraded.
```
Comment (meirege) [inline source/sass/31-molecules/status-message/_status-message.scss:5]: Todo? :)

File: source/sass/61-layouts/_detail-layout.scss [hunk 2]
```diff
@@ -18,4 +18,16 @@
       }
     }
   }
+
+  // @TODO: remove this when the styleguide version is upgraded.
```
Comment (meirege) [inline source/sass/61-layouts/_detail-layout.scss:22]: Todo?

**Review by meirege — APPROVED** [review]
Okay, but shouldn't the 2 @TODO statements be fixed? :)

File: source/sass/61-layouts/_detail-layout.scss (hunk 2 above)
Comment (hstandaert) [reply source/sass/61-layouts/_detail-layout.scss:22]: They're theming overrides for bugs that are still present in the styleguide. I could extract them into a separate file as well, if you find that more clear? 🤔

**Pull request merged**